### PR TITLE
[11.0][MIG] purchase_date_planned_manual

### PR DESCRIPTION
**Pull request by LoisRForgeFlow**
Migration to v11.

### CODE REVIEW

File: purchase_date_planned_manual/tests/test_date_planned_manual.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            'type': 'product',
+            'route_ids': [(4, route_buy_id)],
+            'seller_ids': [(4, supplier_info_id)],
+        })
+        self.product_2 = self.env['product.product'].create({
+            'name': 'Test product 2',
+            'type': 'product',
+        })
+        # Create purchase order and dates
+        self.purchase_order = self.env['purchase.order'].create({
```
Comment (nikul-serpentcs) [inline purchase_date_planned_manual/tests/test_date_planned_manual.py]: IMO `self.env['purchase.order']` replace to `self.po_model`

File: purchase_date_planned_manual/tests/test_date_planned_manual.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        route_buy_id = self.env.ref('purchase.route_warehouse0_buy').id
+        self.partner = self.env['res.partner'].create({
+            'name': 'Test supplier',
+            'supplier': True,
+        })
+        supplier_info_id = self.env['product.supplierinfo'].create({
+            'name': self.partner.id,
+            'delay': 2,
+        }).id
+        self.product_1 = self.env['product.product'].create({
```
Comment (nikul-serpentcs) [inline purchase_date_planned_manual/tests/test_date_planned_manual.py]: IMO, here product use multiple time, so you can create global variable

**Review by nikul-serpentcs — COMMENTED** [review]
Minor change

File: purchase_date_planned_manual/tests/test_date_planned_manual.py (hunk 1 above)
Comment (LoisRForgeFlow) [reply purchase_date_planned_manual/tests/test_date_planned_manual.py]: Done

**Comment by LoisRForgeFlow** [comment]
@nikul-serpentcs Please, update your review.

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Review by HviorForgeFlow — APPROVED** [review]
Code + functional review. It works as expected. :+1:

**Review by MiquelRForgeFlow — APPROVED** [review]
LGTM

**Pull request merged**